gcovr/configuration.py

```diff
@@ -587,6 +599,14 @@ def choose(_cls, choices, default=None):
         default=None,
         const=OutputOrDefault(None),
     ),
+    GcovrConfigOption(
+        "html_css", ["--html-css"],
+        group="output_options",
+        type=check_inputfile,
+        metavar="CSS",
+        help="Use given CSS stylesheet for the HTML report. Default is '{default!s}'.",
+        default=None
```

Comment:
If default is None, the current help message might be confusing. Depending on the exact role of this style sheet, the message should possibly say

> Override the default style sheet for the HTML report.

Reply:
Ok.